recipes/semver200/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conan import ConanFile
+from conan.tools.files import get, patch
+from conans.tools import check_min_cppstd
+from conans import CMake
+
+required_conan_version = ">=1.47.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```
check_min_cppstd requirement